graph-db/serverless/src/utils/cmr/fetchPageFromCMR.js

```diff
@@ -40,82 +40,76 @@ export const fetchPageFromCMR = async ({
     requestHeaders['CMR-Search-After'] = searchAfter
   }
 
-  let fetchUrl = `${process.env.CMR_ROOT}/search/collections.json?page_size=${process.env.PAGE_SIZE}`
+  const { env } = process
+  const { cmrRoot, pageSize } = env
+
+  let fetchUrl = `${cmrRoot}/search/collections.umm_json?page_size=${pageSize}`
 
   if (providerId !== null) {
     fetchUrl += `&provider=${providerId}`
   }
 
   try {
     if (sqs == null) {
-      sqs = new SQSClient()
+      sqs = new SQSClient(getSqsConfig())
     }
+
     // Send request to CMR
     const cmrCollections = await axios({
       url: fetchUrl,
       method: 'GET',
       headers: requestHeaders
     })
+
     // Iterate over CMR-pages
     const { data, headers } = cmrCollections
     const { 'cmr-search-after': cmrsearchAfter } = headers
 
-    const { feed = {} } = data
-    const { entry = [] } = feed
+    const { items = [] } = data
+
     let currentBatchSize = 0
+
     // Split page array into array with sub-arrays of size 10
```

Comment:
```suggestion
    // Split page array into array with sub-arrays of size 5
```

Reply:
Changed to 5 for testing, rather than update the comment I've reverted the value in the code.